### PR TITLE
Contributing.md edit

### PR DESCRIPTION
**Pull request by JustDoom**
# Description

Changed text and added text saying to make sure use correct folder for language

## Checklist

- [x] My contributions follows the style guidelines of this project
- [x] I have performed a self-review of my own contributions (checked for spelling errors, proper linking, etc)
- [x] I have run linting on my branch before submitting the PR

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,9 +29,10 @@ Before you begin:
 - After that, we may have questions, check back on your PR to keep up with the conversation.
 
 ### What goes where
```
Comment (Silverarmor) [inline CONTRIBUTING.md:31]: Folder Conventions*?

"What goes where" is very generic &  informal

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,9 +29,10 @@ Before you begin:
...
-- The tutorials folder is for anythink showing you how to do, setup or use something.
-- The info folder is for any random information like good anticheat, scoreboard or ChatFilter plugins.
-- The recourses folder is for images for other files, configs etc.
+- The [tutorials folder](tutorials) is for any guides on how to setup/use something.
+- The [info folder](info) is for any information on commands, features etc of plugins, server jars etc.
+- The [resources folder](resources) is for images for other files, configs etc.
```
Comment (Silverarmor) [inline CONTRIBUTING.md:34]: "for images for other files"?

possibly "for images **and** other files"?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -29,9 +29,10 @@ Before you begin:
...
+- Make sure you also put the contribution is in the correct language folder.
 
 ### Your PR is merged
```
Comment (Silverarmor) [inline CONTRIBUTING.md:37]: This doesnt make sense as a heading. Possibly overhaul the file with numbered instructions?

**Review by Silverarmor — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (JustDoom) [reply CONTRIBUTING.md:37]: what do you mean?

Comment (JustDoom) [reply CONTRIBUTING.md:37]: wait the your pr is merged?

Comment (Silverarmor) [reply CONTRIBUTING.md:37]: Yes - The  comment was line 36 / 37.....

Comment (JustDoom) [reply CONTRIBUTING.md:37]: it was there already but what should i change it to then?

Comment (Silverarmor) [reply CONTRIBUTING.md:37]: >  Possibly overhaul the file with numbered instructions?

Comment (JustDoom) [reply CONTRIBUTING.md:37]: i think its fine, and idk what to change it to because i didnt even add it lol

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -28,10 +28,12 @@ Before you begin:
...
 
-### What goes where
-- The tutorials folder is for anythink showing you how to do, setup or use something.
-- The info folder is for any random information like good anticheat, scoreboard or ChatFilter plugins.
-- The recourses folder is for images for other files, configs etc.
+### Folder Conventions
+
+- The [tutorials folder](tutorials) is for any guides on how to setup/use something.
+- The [info folder](info) is for any information on commands, features etc of plugins, server jars etc.
+- The [resources folder](resources) is for images and other files, configs etc.
```
Comment (Silverarmor) [inline CONTRIBUTING.md:35]: This doesn't make sense.  Just leave as
"is for images and other files"

**Review by Silverarmor — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -28,10 +28,12 @@ Before you begin:
...
-### What goes where
-- The tutorials folder is for anythink showing you how to do, setup or use something.
-- The info folder is for any random information like good anticheat, scoreboard or ChatFilter plugins.
-- The recourses folder is for images for other files, configs etc.
+### Folder Conventions
+
+- The [tutorials folder](tutorials) is for any guides on how to setup/use something.
+- The [info folder](info) is for any information on commands, features etc of plugins, server jars etc.
+- The [resources folder](resources) is for images and other files.
+- Make sure you also put the contribution is in the correct language folder.
```
Comment (Silverarmor) [inline CONTRIBUTING.md:36]: Read Line 36 aloud

Comment (Silverarmor) [reply CONTRIBUTING.md:36]: You should be checking your commits before pushing them. Do they make sense?

Comment (JustDoom) [reply CONTRIBUTING.md:36]: i swear i do. ill try to read like 10 times before i commit

**Review by Silverarmor — APPROVED** [review]

**Pull request merged**